src/ansys/mapdl/core/_commands/database/selecting.py

```diff
@@ -20,1853 +20,2065 @@
+        ``KSWP`` field is processed, but the ``Comp``, ``VMIN``, ``VMAX``, and ``VINC`` fields are ignored.
+
+        If ``Item`` = HPT, the command selects only those lines that contain hard points.
+
+        ``Item`` = RADIUS is only valid for lines that are circular arcs.
+
+        :ref:`lsel` is valid in any processor.
+
+        For selections  based on non-integer numbers (coordinates, results, etc.), items that are
+        within the range ``VMIN`` - ``Toler`` and ``VMAX`` + ``Toler`` are selected. The default tolerance
+        ``Toler`` is based on the relative values of ``VMIN`` and ``VMAX`` as follows:
+
+        * If ``VMIN`` = ``VMAX``, ``Toler`` = 0.005 x ``VMIN``.
+
+        * If ``VMIN`` = ``VMAX`` = 0.0, ``Toler`` = 1.0E-6.
+
+        * If ``VMAX`` ≠ ``VMIN``, ``Toler`` = 1.0E-8 x ( ``VMAX`` - ``VMIN`` ).
+
+        Use the `SELTOL
+        <https://ansyshelp.ansys.com/Views/Secured/corp/v232/en/ans_cmd/Hlp_C_SELTOL.html#SELTOL.menupath>`_
+        :ref:`seltol` command to override this default and specify ``Toler`` explicitly.
+
+        .. _lsel.tab.1:
+
+        **LSEL - Valid Item and Component Labels**
+
+        .. flat-table:: Valid Item and Component Labels :ref:`lsel`, ``Type, Item, Comp, VMIN, VMAX, VINC, KSWP``
+           :header-rows: 1
+
+           * - Item
+             - Comp
+             - Description
+           * - LINE
+             -
+             - Line number.
+           * - EXT
+             -
+             - Line numbers on exterior of selected area (ignore remaining fields).
+           * - LOC
+             - X,Y,Z
+             - X, Y, or Z center location in the active coordinate system.
+           * - TAN1
+             - X,Y,Z
+             - Unit vector component of outward tangent at beginning of line.
+           * - TAN2
+             - X,Y,Z
+             - Unit vector component of outward tangent at end of line.
+           * - NDIV
+             -
+             - Number of divisions within the line.
+           * - SPACE
+             -
+             - Spacing ratio of line divisions.
+           * - MAT
+             -
+             - Material number associated with the line.
+           * - TYPE
+             -
+             - Element type number associated with the line.
+           * - REAL
+             -
+             - Real constant set number associated with the line.
+           * - ESYS
+             -
+             - Element coordinate system associated with the line.
+           * - SEC
+             -
+             - Cross section ID number. ( :ref:`secnum` )
+           * - LENGTH
+             -
+             - Length of the line.
+           * - RADIUS
+             -
+             - Radius of the line.
+           * - HPT
+             -
+             - Line number (selects only lines with associated hard points).
+           * - LCCA
+             -
+             - Concatenated lines (selects only lines that were created by concatenation ( :ref:`lccat` )).
```

Comment:
where we can see this rendered? there was an online website with this, no?

Reply:
https://cautious-adventure-g6vqq8v.pages.github.io/version/dev/database/_autosummary/ansys.mapdl.core._commands.database.selecting.Selecting.lsel.html#lsel

Reply:
Is it fine by you @germa89? If so, I will merge this PR